docs: added video to gauge panel doc

srperf opened the pull request:
**What is this feature?**

Added a brief description and link to the video explaining the Gauge panel visualization.

**Why do we need this feature?**

It improves the docs' user experience for visual learners by displaying a video.

**Who is this feature for?**

Everyone looking into mastering Grafana Gauge panels.

**Which issue(s) does this PR fix?**:

None

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

srperf committed:
Added Data and fixed location mode

srperf committed:
Updated Gauge panel doc adding video link

Added a short description and video link to the new beginners video

imatwawana commented on docs/sources/panels-visualizations/visualizations/geomap/index.md:
```diff
@@ -98,6 +98,10 @@ The initial view configures how the geomap renders when the panel is first loade
 
+### Share view
+
+The Share view option allows users to link multiple map panels' movement and zoom actions within the same dashboard. The map panels that have this option activated will act in tandem when one of them is moved or zoomed, leaving the deactivated ones independent. _Note: this may require a dashboard reload._
```
```suggestion
The **Share view** option allows you to link the movement and zoom actions of multiple map visualizations within the same dashboard. The map visualizations that have this option enabled act in tandem when one of them is moved or zoomed, leaving the other ones independent.

{{< admonition type="note" >}}
You might need to reload the dashboard for this feature to work.
{{< /admonition >}}
```

imatwawana commented on docs/sources/panels-visualizations/visualizations/geomap/index.md:
```diff
@@ -156,9 +160,13 @@ The layer controls allow you to create layers, change their name, reorder and de
-### Location
+### Data
+
+Geomaps need a source of geographical data gathered from a data source query which can return multiple data sets. By default Grafana picks the first data set, this field allows users to pick other data sets if the query returns more than one.
```
```suggestion
Geomaps need a source of geographical data gathered from a data source query which can return multiple datasets. By default Grafana picks the first dataset, but this drop-down allows you to pick other datasets if the query returns more than one.
```

imatwawana commented on docs/sources/panels-visualizations/visualizations/geomap/index.md:
```diff
@@ -156,9 +160,13 @@ The layer controls allow you to create layers, change their name, reorder and de
+### Location mode
 
-Geomaps need a source of geographical data. This data comes from a database query, and there are four mapping options for your data.
+There are four options to map the data returned by the selected query.
```
```suggestion
There are four options to map the data returned by the selected query:
```

imatwawana reviewed (commented):
Looks great! I've made a few wording and style edits.

srperf committed:
Isabel edits for better reading

Co-authored-by: Isabel Matwawana <76437239+imatwawana@users.noreply.github.com>

srperf committed:
Geomap updates

Not sure why this ended in this PR but I will check the Geomap one I am working on :P

Co-authored-by: Isabel Matwawana <76437239+imatwawana@users.noreply.github.com>

srperf committed:
Geomap updates

Same, not sure why this part ended in this PR but more coming :)

Co-authored-by: Isabel Matwawana <76437239+imatwawana@users.noreply.github.com>
